codegenerator/cli/templates/dynamic/codegen/package.json.hbs

```diff
@@ -51,7 +51,7 @@
     "postgres": "3.4.1",
     "prom-client": "15.0.0",
     "react": "18.2.0",
-    "rescript": "11.1.0",
+    "rescript": "11.1.3",
```

Comment:
The upgrade was needed because I came across a bug that appears to be fixed now. See this example: https://rescript-lang.org/try?version=v11.1.0&code=DYUwLgBAxgTiCGYQCFFQBYEkB2SbfmAgF4IAKAPwFt4APVMDAZQEsAvEAGggoHNwAciFpgAlCQB8EAN4AoCBFCQARmnQkIAbQC68xeAhwoigPYmADhrLjiEvQpYAzCKsboAtBICCMGPACeAHSg2Lxg6hKkNPRqrBwy9gouaokAvhAgwADOIAlJSVkA7ixuEPxgQiLWefkQAD4QAibYubbJbokKDUwmVCBkZCUgVNzmFphIVADyjo4AigCuIEuiNna1SWPmE8Mz80tL1p1JrhieAFJZgT5+-gBMgeYLWeiDk6KeLLzYJnDHCsAzOYjhtUmk9ICLEcwUA

It tries to set `_param=undefined` and breaks at runtime

Reply:
I'll do a release for Fuel indexer tomorrow to prevent it from breaking